packages/design/src/DataTable/Table.jsx

```diff
@@ -149,12 +149,13 @@ class Column extends React.Component {
 }
 
 const Cell = props => {
-  const { isHeader, children, align, style, className } = props;
+  const { isHeader, children, align, style, className, title } = props;
   const childProps = {
     children,
     align,
     style,
     className,
+    title,
```

Comment:
Where is it used? `title`

Reply:
here, it's to enable on hover show full text: https://github.com/gravitational/webapps.e/pull/31/files#diff-bd72ddd2c5cda80ddbf61e0b7a50791c0f022afd31eeb78517fdbab8df343cc2R180